Src/LiquidProjections.NEventStore/NEventStoreAdapter.cs

```diff
@@ -299,7 +335,7 @@ private void StartPreloadingNextPage(long previousCheckpoint, string subscriptio
                         $"Failed loading transactions after checkpoint {previousCheckpoint} from NEventStore",
                         exception);
 
-                return new Page(previousCheckpoint, new List<Transaction>());
+                throw;
```

Comment:
Is probably fine. In the future we may even decide to add some progressively increasing delay.